src/features/barn/components/AnimalBounties.tsx

```diff
@@ -0,0 +1,299 @@
+                    >
+                      {deal.coins}
+                    </Label>
+                  )}
+
+                  {getKeys(deal.items ?? {}).map((name) => {
+                    return (
+                      <Label
+                        key={name}
+                        type="warning"
+                        icon={ITEM_DETAILS[name].image}
+                        className={"absolute -bottom-2 text-center p-1 "}
+                        style={{
+                          left: `${PIXEL_SCALE * -3}px`,
+                          right: `${PIXEL_SCALE * -3}px`,
+                          width: `calc(100% + ${PIXEL_SCALE * 6}px)`,
+                          height: "25px",
+                        }}
+                      >
+                        {deal.items?.[name]}
+                      </Label>
+                    );
+                  })}
+                </ButtonPanel>
+              </div>
+            );
+          })}
+        </div>
+      </div>
+    </CloseButtonPanel>
+  );
+};
+
+export const AnimalDeal: React.FC<{
+  deal: BountyRequest;
+  animal: Animal;
+  onClose: () => void;
+  onSold: () => void;
+}> = ({ deal, animal, onClose, onSold }) => {
+  const { gameService } = useContext(Context);
+
+  const { t } = useAppTranslation();
+  const sell = () => {
+    gameService.send("animal.sold", {
+      requestId: deal.id,
+      animalId: animal.id.toString(),
+    });
+
+    onSold();
+  };
+
+  if (!deal || !animal) {
+    return null;
+  }
+
+  return (
+    <Panel>
+      <div className="p-2">
+        <div className="mb-2 flex flex-wrap">
+          <Label
+            type="default"
+            icon={ITEM_DETAILS[animal.type].image}
+            className="mr-2"
+          >
+            {`Lvl ${getAnimalLevel({ animal })} ${animal.type}`}
+          </Label>
+          {!!deal.coins && (
+            <Label type="warning" icon={SUNNYSIDE.ui.coinsImg}>
+              {deal.coins}
+            </Label>
+          )}
+
+          {getKeys(deal.items ?? {}).map((name) => {
+            <Label key={name} type="warning" icon={ITEM_DETAILS[name].image}>
+              {deal.items?.[name]}
+            </Label>;
+          })}
+        </div>
+
+        <p>
+          {deal.coins
+            ? t("bounties.sell.coins", { amount: deal.coins })
+            : t("bounties.sell.items", {
+                amount: getKeys(deal.items ?? {})
+                  .map((name) => `${deal.items?.[name]} x ${name}`)
+                  .join(" - "),
+              })}
+        </p>
+      </div>
+      <div className="flex">
+        <Button className="mr-1" onClick={onClose}>
+          {t("cancel")}
+        </Button>
+        <Button onClick={sell}>{t("confirm")}</Button>
+      </div>
+    </Panel>
+  );
+};
+
+export const ExchangeHud: React.FC<{
+  deal: BountyRequest;
+  onClose: () => void;
+}> = ({ deal, onClose }) => {
+  const { t } = useAppTranslation();
+  return (
+    <HudContainer>
```

Comment:
When clicking on a chicken inside of this hud it triggers the `animal.fed` action from the `Chicken` component. So what happens is: 

`feed` -> `sell`